web-wallet/src/routes/(welcome)/setup/restore/+page.svelte

```diff
@@ -45,7 +45,7 @@
   /** @type {string[]} */
   let mnemonicPhrase = $mnemonicPhraseResetStore;
 
-  let blockHeight = 0;
+  let blockHeight = "0";
```

Comment:
Why don't we just have this as `0n`? I don't get the need for conversion to string and then back to BigInt.

Reply:
because the string is the value that goes into the textbox

Reply:
I would still probably contain the logic to convert to bigint only in the component where the text field is and use bigint everywhere else. But it's a minor comment anyways.